.github/workflows/scripts/add-codeowners-to-pr.sh

```diff
@@ -29,9 +29,9 @@ main () {
     # is only available through the API. To cut down on API calls to GitHub,
     # we use the latest reviews to determine which users to filter out.
     JSON=$(gh pr view "${PR}" --json "files,author,latestReviews" | tr -dc '[:print:]' | sed -E 's/\\[a-z]//g')
-    AUTHOR=$(printf "${JSON}"| jq -r '.author.login')
-    FILES=$(printf "${JSON}"| jq -r '.files[].path')
-    REVIEW_LOGINS=$(printf "${JSON}"| jq -r '.latestReviews[].author.login')
+    AUTHOR=$(echo "${JSON}"| jq -r '.author.login')
+    FILES=$(echo "${JSON}"| jq -r '.files[].path')
```

Comment:
Sorry for jumping in late here, but since `printf` won't print a newline by default and `echo` will, should we add `-n` here? If it doesn't make a difference no need to add extra flags, I'm just not sure.

Reply:
If u wish i can just add `-n`, to get the output we had before it might be the easiest way?

Reply:
fixed it for the important ones, which aren't just text-displaying